src/speechly/client.ts

```diff
@@ -87,7 +89,7 @@ export class Client {
       new WebsocketClient(
         options.loginUrl ?? defaultLoginUrl,
         options.apiUrl ?? defaultApiUrl,
-        options.language,
+        options.language ?? defaultLanguage,
```

Comment:
Since we already store validated input to the `language` variable, I think it makes sense to use it here:
```suggestion
        language
```